test/unittests.py

```diff
@@ -2,7 +2,7 @@
 from rhino_shapely_interop.rhino_wrappers import RhPnt, RhCurv
 from rhino_shapely_interop.importers import RhImporter
 import unittest
-from shapely.geometry import Point, asMultiPoint, Polygon, LineString, MultiPoint
+from shapely.geometry import Point, MultiPoint, Polygon, LineString, MultiPoint
```

Comment:
```suggestion
from shapely.geometry import Point, Polygon, LineString, MultiPoint
```
Minor catch. `MultiPoint` used twice.